src/components/Tabs/Tab/__tests__/tab-tests.jest.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { render, fireEvent, screen } from "@testing-library/react";
+import Tab from "../Tab";
+
```

Comment:
please remove not needed spaces and change test name according to component name